pkg/config/datafileprojectconfig/mappers/condition_trees_test.go

```diff
@@ -159,3 +159,28 @@ func TestBuildConditionTreeSimpleAudienceCondition(t *testing.T) {
 	}
 	assert.Equal(t, expectedConditionTree, conditionTree)
 }
+
+func TestBuildConditionTreeWithLeafNode(t *testing.T) {
+	conditionString := "{ \"type\": \"custom_attribute\", \"name\": \"s_foo\", \"match\": \"exact\", \"value\": \"foo\" }"
+	var conditions interface{}
+	json.Unmarshal([]byte(conditionString), &conditions)
+	conditionTree, err := buildConditionTree(conditions)
+	if err != nil {
```

Comment:
Instead of doing this can you assert no error? Seems a little more straightforward

Reply:
agree

Reply:
done 👍